be/test/storage/persistent_index_test.cpp

```diff
@@ -89,4 +91,35 @@ PARALLEL_TEST(PersistentIndexTest, test_mutable_index) {
     ASSERT_EQ(upsert_not_found.key_idxes.size(), expect_not_found);
 }
 
+PARALLEL_TEST(PersistentIndexTest, test_mutable_flush_to_immutable) {
+    using Key = uint64_t;
+    int N = 200000;
+    vector<Key> keys(N);
+    vector<IndexValue> values(N);
+    for (int i = 0; i < N; i++) {
+        char* dst = (char*)&keys[i];
+        snprintf(dst, 8, "%07d", i);
```

Comment:
use fmt::format instead, the performance is better to convert int to string.

Reply:
fixed